### PR TITLE
Deal with vmodl.fault.managed object not found in get vcenter object by name

### PR DESCRIPTION
**Pull request by daniel-kg**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -2,7 +2,8 @@
 
 
 setup(
-    version='0.0.0',
+
+    version='0.0.1',
```
Comment (graingert) [inline setup.py]: I'll update this version as part of the release process

File: vcdriver/helpers.py [hunk 2]
```diff
@@ -60,10 +60,18 @@ def get_vcenter_object_by_name(connection, object_type, name):
     """
     content = connection.RetrieveContent()
     view = content.viewManager.CreateContainerView
-    objects = [
-        obj for obj in view(content.rootFolder, [object_type], True).view
-        if hasattr(obj, 'name') and obj.name == name
-    ]
+    objects = []
+    for obj in view(content.rootFolder, [object_type], True).view:
```
Comment (graingert) [inline vcdriver/helpers.py]: can you add a test for this?

Comment (daniel-kg) [reply vcdriver/helpers.py]: I extended the current one to cover this branch

Comment (graingert) [reply vcdriver/helpers.py]: oh I just saw it

File: vcdriver/helpers.py [hunk 3]
```diff
@@ -60,10 +60,18 @@ def get_vcenter_object_by_name(connection, object_type, name):
     """
     content = connection.RetrieveContent()
     view = content.viewManager.CreateContainerView
-    objects = [
```
Comment (graingert) [inline vcdriver/helpers.py]: I'd like to keep this as close to the existing code as possible:

eg including the hasattr

```
def name_matches(obj):
    if hasattr(obj, 'name'):
        try:
            return obj.name == name
        except vmodl.fault.ManagedObjectNotFound:
            pass
    return False

objects = [
    obj for obj in view(content.rootFolder, [object_type], True).view
    if name_matches(obj)
]
```

File: setup.py [hunk 4]
```diff
@@ -2,6 +2,7 @@
 
 
 setup(
+
```
Comment (graingert) [inline setup.py]: ws

**Pull request merged**